index.html

```diff
@@ -0,0 +1,84 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Nemwel Boniface">
+    <meta name="description" content="This is my personal portfolio site that I will share to all potential employers to showcase my skills as a developer">
+
+    <!--google Fonts roboto, crete-round-->
+    
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+
+    <!-- font awesome icons -->
+
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <title>Nemwel Portfolio</title>
+
+    <!--Page stylesheets-->
+    <link rel="stylesheet" href="stylesheets/landing.css">
+
+    <!--Smooth scroll script-->
+       
+    <script src="https://cdn.jsdelivr.net/gh/cferdinandi/smooth-scroll/dist/smooth-scroll.polyfills.min.js"></script>
+
+</head>
+<body>
+    <header class="header">
+        <div class="topHeaderBar">
+            <div class="time-left">
+                <h2>9:41</h2>
+            </div>
+
+            <div class="icons-right">
+                <i class="fa fa-signal" aria-hidden="true"></i>
+                <i class="fa fa-wifi" aria-hidden="true"></i>
+                <i class="fa fa-battery-full" aria-hidden="true"></i> 
+            </div>
+        </div>
+
+        <nav class="topnavBar">
+            <div class="navleft">
+                <h2>Nemwel Boniface</h2>
+            </div>
+
+            <div class="navright">
+                <i class="fa fa-bars" aria-hidden="true"></i>
+            </div>
+        </nav>
+    </header>
+
+    <main class="main">
+        <article class="authordetails">
+            <div class="authorheader">
+                <h2>Hey There. I'm Nemwel</h2>
+            </div>
+
+            <div class="authoroccupation">
+                <p>I’am a Software Developer</p>
+            </div>
+
+            <div class="authorexplanation">
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+            </div>
+
+            <div class="authorsocials">
+                <img src="images/github.png" alt="github logo">
+                <div class="imagejoin">
+                    <img src="images/linkedin-i.png" alt="linkedin logo">
+                    <img src="images/linkedin-n.png" alt="linkedin logo">
+                </div>
+                <img src="images/cheers.png" alt="cheers logo">
+                <img src="images/tiwitter.png" alt="twitter logo">
+                <img src="images/M.png" alt="m logo">
+            </div>
```

Comment:
- [ ] Good job importing the icons! However, their shape is a little off.  Let's kindly correct them.

Your icons (I corrected the first one as an example):

![image](https://user-images.githubusercontent.com/57421823/148378037-ae5c23a9-b8b3-4ebc-b832-c701a5ab9212.png)

The Figma's:
![image](https://user-images.githubusercontent.com/57421823/148378330-58c6a7c9-84da-417d-baea-ed8f4970179c.png)


This can be achieved by adjusting their width and height!  The exact numbers in pixels are in the Figma template.

That way, the icons will look as intended in the design.


- [ ] Your icons lack a <a> link, please kindly add it.